src/main/java/org/broadinstitute/dsm/export/ExportToES.java

```diff
@@ -126,20 +127,38 @@ public static void checkWorkflowNamesAndExport(List<String> workFlowColumnNames,
         DDPInstance ddpInstance = DDPInstance.getDDPInstanceById(instanceId);
         for (ParticipantDataDto participantData: allParticipantData) {
             String data = participantData.getData();
-            if (data != null) {
-                JsonObject dataJsonObject = gson.fromJson(data, JsonObject.class);
-                for (Map.Entry<String, JsonElement> entry: dataJsonObject.entrySet()) {
-                    if (workFlowColumnNames.contains(entry.getKey())) {
-                        ElasticSearchUtil.writeWorkflow(ddpInstance, participantData.getDdpParticipantId(),
-                                entry.getKey(), entry.getValue().getAsString());
-                    }
+            if (data == null) {
+                continue;
+            }
+            JsonObject dataJsonObject = gson.fromJson(data, JsonObject.class);
```

Comment:
What's the benefit of using `JsonObject` here and make things complicated than using `Map<String, String>` directly since `data` column in `ddp_participant_data` has `varchar` type and everything there will be strings, you have `JSON` already in `data` and here you are converting `JSON` to again `JSON` object